website/common/locales/en/front.json

```diff
@@ -15,7 +15,17 @@
   "emailNewPass": "Email a Password Reset Link",
   "forgotPasswordSteps": "Enter the email address you used to register your Habitica account.",
   "sendLink": "Send Link",
-  "featuredIn": "Featured in",
+  "evagantzQuote": "My very first dentist appointment where the hygienist was actually excited about my flossing habits. Thanks [Habitica]!",
```

Comment:
@jbusa22 it's all looking good, but I don't understand why these strings were added here, they're not used anywhere

Reply:
I've fixed this